lib/scrabble.rb

```diff
@@ -15,4 +17,24 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    if word.nil?
+      @score
+    else
+      letters = word.upcase.split("")
+
+      letters.each do |letter|
```

Comment:
Would love to see you push to use something like reduce here or map & sum instead of each.